Client/sdk/game/CCamera.h

```diff
@@ -95,6 +95,13 @@ enum eVehicleCamMode
     MODE_CINEMATIC
 };
 
+enum class ePedCamMode
+{
+    MODE_CLOSE,
+    MODE_MIDDLE,
+    MODE_FAR,
+};
```

Comment:
enum classes don't need a `MODE_` prefix, because they can only be accessed like `ePedCamMode::MODE_CLOSE`.

I don't know if we want `ePedCamMode::Close` or `ePedCamMode::CLOSE`, so you can just pick one